stage6/08-eepromutil/00-run.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+
+on_chroot << EOF
+
+mkdir -p "home/${FIRST_USER_NAME}"
+pushd "home/${FIRST_USER_NAME}"
+
+[ -d hats" ] || {
```

Comment:
is this on purpose? or did you meant `[ -d "hats" ]`?

Reply:
missed that. will add it.

Reply:
in the future if we know we are using bash, we can just use `[[ ]]` and safely forget about the quotes